src/foam/nanos/crunch/UserReferencedPayload.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ foam.CLASS({
+  package: 'foam.nanos.crunch',
+  name: 'UserReferencedPayload',
+  documentation: `
+    Capability user related payload class.
+    Attaches listeners to wizardlets if defined.
+    Useful for extending payload classes that would potentially be used by
+    capable objects requiring user specific logic such as copying user values
```

Comment:
so capable objects here you mean both ucj and capable, or just capable (where the payload is stored on the object)?

Reply:
Capable objects